client/src/main/java/com/connectifier/xeroclient/XeroClient.java

```diff
@@ -128,7 +128,8 @@ protected ResponseType get(String endPoint, Date modifiedAfter, Map<String,Strin
   protected ResponseType put(String endPoint, JAXBElement<?> object) {
     OAuthRequest request = new OAuthRequest(Verb.PUT, BASE_URL + endPoint);
     String contents = marshallRequest(object);
-    request.addPayload(contents);
+    request.setCharset("UTF-8"); // set character set to UTF-8
```

Comment:
do you mind leaving off the comment? it's pretty redundant and doesn't add much. other than that, this looks good and i can merge